### PR TITLE
test public workspace google group sync

### PR DESCRIPTION
**Pull request by dvoet**
Ticket: <Link to Jira ticket>
<Put notes here to help reviewer understand this PR>

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

**Comment by dvoet** [comment]
jenkins retest

**Review by Qi77Qi — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/SamApiSpec.scala [hunk 1]
```diff
@@ -539,6 +539,40 @@ class SamApiSpec extends FreeSpec with BillingFixtures with Matchers with ScalaF
...
+      Sam.user.createResource(resourceTypeName, CreateResourceRequest(resourceId, Map(ownerPolicyName -> AccessPolicyMembership(Set(user1.email), Set.empty, Set("owner")), readerPolicyName -> AccessPolicyMembership(Set.empty, Set.empty, Set("reader"))), Set.empty))(user1AuthToken)
+      register cleanUp Sam.user.deleteResource(resourceTypeName, resourceId)(user1AuthToken)
+
+      val policies = Sam.user.listResourcePolicies(resourceTypeName, resourceId)(user1AuthToken)
+      val ownerPolicy = policies.filter(_.policyName equals ownerPolicyName).last
+      val readerPolicy = policies.filter(_.policyName equals readerPolicyName).last
+
+      Sam.user.syncResourcePolicy(resourceTypeName, resourceId, ownerPolicyName)(user1AuthToken)
+
+      awaitAssert(
```
Comment (Qi77Qi) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/SamApiSpec.scala:563]: would be nice to add comment saying this line is to validate actually constrained resource and the other assert is to validate constrainable but not actually constrained resource.

also, shall we add a test for public constrained resource?

Comment (dvoet) [reply automation/src/test/scala/org/broadinstitute/dsde/test/api/SamApiSpec.scala:563]: public constrained resources are not allowed which is tested in unit tests

**Comment by dvoet** [comment]
jenkins restart - cromwell reversion issue

**Comment by dvoet** [comment]
I meant jenkins retest

**Pull request closed without merging**